From d1d13cf7b6dffa7e9b21c7b6a0edea137677ce83 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Tue, 28 Nov 2017 06:06:05 -0800
Subject: [PATCH 438/456] Branching for 5.14.1-SNAPSHOT on Tue Nov 28 06:05:46
 PST 2017

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '514'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hbase.git (push)
origin	git@github.sf.cloudera.com:CDH/hbase.git (push)
---
 hbase-annotations/pom.xml                          |    2 +-
 hbase-assembly/pom.xml                             |    2 +-
 hbase-checkstyle/pom.xml                           |    4 ++--
 hbase-client/pom.xml                               |    2 +-
 hbase-common/pom.xml                               |    2 +-
 hbase-examples/pom.xml                             |    2 +-
 hbase-external-blockcache/pom.xml                  |    2 +-
 hbase-hadoop-compat/pom.xml                        |    2 +-
 hbase-hadoop2-compat/pom.xml                       |    2 +-
 hbase-it/pom.xml                                   |    2 +-
 hbase-prefix-tree/pom.xml                          |    2 +-
 hbase-procedure/pom.xml                            |    2 +-
 hbase-protocol/pom.xml                             |    2 +-
 hbase-resource-bundle/pom.xml                      |    2 +-
 hbase-rest/pom.xml                                 |    2 +-
 hbase-rsgroup/pom.xml                              |    2 +-
 hbase-server/pom.xml                               |    2 +-
 hbase-shaded/hbase-shaded-check-invariants/pom.xml |    2 +-
 hbase-shaded/hbase-shaded-client/pom.xml           |    2 +-
 hbase-shaded/hbase-shaded-server/pom.xml           |    2 +-
 hbase-shaded/pom.xml                               |    2 +-
 hbase-shell/pom.xml                                |    2 +-
 hbase-spark/pom.xml                                |    2 +-
 hbase-testing-util/pom.xml                         |    2 +-
 hbase-thrift/pom.xml                               |    2 +-
 pom.xml                                            |    4 ++--
 26 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/hbase-annotations/pom.xml b/hbase-annotations/pom.xml
index 4b45517..1b3060c 100644
--- a/hbase-annotations/pom.xml
+++ b/hbase-annotations/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-assembly/pom.xml b/hbase-assembly/pom.xml
index eabe258..8c4cf99 100644
--- a/hbase-assembly/pom.xml
+++ b/hbase-assembly/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-assembly</artifactId>
diff --git a/hbase-checkstyle/pom.xml b/hbase-checkstyle/pom.xml
index dea9a3f..ff70487 100644
--- a/hbase-checkstyle/pom.xml
+++ b/hbase-checkstyle/pom.xml
@@ -24,14 +24,14 @@
 <modelVersion>4.0.0</modelVersion>
 <groupId>org.apache.hbase</groupId>
 <artifactId>hbase-checkstyle</artifactId>
-<version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+<version>1.2.0-cdh5.14.1-SNAPSHOT</version>
 <name>Apache HBase - Checkstyle</name>
 <description>Module to hold Checkstyle properties for HBase.</description>
 
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-client/pom.xml b/hbase-client/pom.xml
index 26632f5..e13123b 100644
--- a/hbase-client/pom.xml
+++ b/hbase-client/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-common/pom.xml b/hbase-common/pom.xml
index eabdc91..a4b9cff 100644
--- a/hbase-common/pom.xml
+++ b/hbase-common/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-examples/pom.xml b/hbase-examples/pom.xml
index ff9c0a3..19df6bf 100644
--- a/hbase-examples/pom.xml
+++ b/hbase-examples/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-examples</artifactId>
diff --git a/hbase-external-blockcache/pom.xml b/hbase-external-blockcache/pom.xml
index 60bf66b..ff91e20 100644
--- a/hbase-external-blockcache/pom.xml
+++ b/hbase-external-blockcache/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-external-blockcache</artifactId>
diff --git a/hbase-hadoop-compat/pom.xml b/hbase-hadoop-compat/pom.xml
index e308a1c..e6028bd 100644
--- a/hbase-hadoop-compat/pom.xml
+++ b/hbase-hadoop-compat/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-hadoop2-compat/pom.xml b/hbase-hadoop2-compat/pom.xml
index 59120db..2ef130f 100644
--- a/hbase-hadoop2-compat/pom.xml
+++ b/hbase-hadoop2-compat/pom.xml
@@ -21,7 +21,7 @@ limitations under the License.
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-it/pom.xml b/hbase-it/pom.xml
index 5ae781c..5ed353c 100644
--- a/hbase-it/pom.xml
+++ b/hbase-it/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-prefix-tree/pom.xml b/hbase-prefix-tree/pom.xml
index baf7bbc..c7dc6ba 100644
--- a/hbase-prefix-tree/pom.xml
+++ b/hbase-prefix-tree/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-procedure/pom.xml b/hbase-procedure/pom.xml
index 4f8dd11..46c2e4e 100644
--- a/hbase-procedure/pom.xml
+++ b/hbase-procedure/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-protocol/pom.xml b/hbase-protocol/pom.xml
index 0620e0a..09417c0 100644
--- a/hbase-protocol/pom.xml
+++ b/hbase-protocol/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-resource-bundle/pom.xml b/hbase-resource-bundle/pom.xml
index c2a3d11..8165488 100644
--- a/hbase-resource-bundle/pom.xml
+++ b/hbase-resource-bundle/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-rest/pom.xml b/hbase-rest/pom.xml
index bb7b314..dcda403 100644
--- a/hbase-rest/pom.xml
+++ b/hbase-rest/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-rest</artifactId>
diff --git a/hbase-rsgroup/pom.xml b/hbase-rsgroup/pom.xml
index 3a155cc..29d0b73 100644
--- a/hbase-rsgroup/pom.xml
+++ b/hbase-rsgroup/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-server/pom.xml b/hbase-server/pom.xml
index 5e5d164..bbdebfe 100644
--- a/hbase-server/pom.xml
+++ b/hbase-server/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-server</artifactId>
diff --git a/hbase-shaded/hbase-shaded-check-invariants/pom.xml b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
index 73da5e1..9b2b315 100644
--- a/hbase-shaded/hbase-shaded-check-invariants/pom.xml
+++ b/hbase-shaded/hbase-shaded-check-invariants/pom.xml
@@ -16,7 +16,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>../..</relativePath>
   </parent>
   <artifactId>hbase-shaded-check-invariants</artifactId>
diff --git a/hbase-shaded/hbase-shaded-client/pom.xml b/hbase-shaded/hbase-shaded-client/pom.xml
index dcc0a0f..908694c 100644
--- a/hbase-shaded/hbase-shaded-client/pom.xml
+++ b/hbase-shaded/hbase-shaded-client/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-client</artifactId>
diff --git a/hbase-shaded/hbase-shaded-server/pom.xml b/hbase-shaded/hbase-shaded-server/pom.xml
index 70902ac..8752a6d 100644
--- a/hbase-shaded/hbase-shaded-server/pom.xml
+++ b/hbase-shaded/hbase-shaded-server/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-server</artifactId>
diff --git a/hbase-shaded/pom.xml b/hbase-shaded/pom.xml
index 4203835..161f79c 100644
--- a/hbase-shaded/pom.xml
+++ b/hbase-shaded/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded</artifactId>
diff --git a/hbase-shell/pom.xml b/hbase-shell/pom.xml
index 19b4409..a7879b3 100644
--- a/hbase-shell/pom.xml
+++ b/hbase-shell/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-shell</artifactId>
diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index f9a3ae3..fa13fcf 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -30,7 +30,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-spark</artifactId>
diff --git a/hbase-testing-util/pom.xml b/hbase-testing-util/pom.xml
index c8b3b11..362b5fa 100644
--- a/hbase-testing-util/pom.xml
+++ b/hbase-testing-util/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+        <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-testing-util</artifactId>
diff --git a/hbase-thrift/pom.xml b/hbase-thrift/pom.xml
index 88ddd12..942044d 100644
--- a/hbase-thrift/pom.xml
+++ b/hbase-thrift/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+    <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-thrift</artifactId>
diff --git a/pom.xml b/pom.xml
index 25b84b7..7c359b4 100644
--- a/pom.xml
+++ b/pom.xml
@@ -32,14 +32,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.14.0-SNAPSHOT</version>
+    <version>5.14.1-SNAPSHOT</version>
     <relativePath/>
     <!-- no parent resolution -->
   </parent>
   <groupId>org.apache.hbase</groupId>
   <artifactId>hbase</artifactId>
   <packaging>pom</packaging>
-  <version>1.2.0-cdh5.14.0-SNAPSHOT</version>
+  <version>1.2.0-cdh5.14.1-SNAPSHOT</version>
   <name>Apache HBase</name>
   <description>
     Apache HBase is the Hadoop database. Use it when you need
-- 
1.7.9.5

